src/libPMacc/include/algorithms/math/floatMath/exp.tpp

```diff
@@ -56,6 +56,21 @@ struct Log<float>
     }
 };
 
+template<>
+struct Log10<float>
+{
+    typedef float result;
+
+    HDINLINE float operator( )(const float& value )
```

Comment:
Space is missing after the parentheses. `HDINLINE float operator( )( const....)`